future-apps/payroll/contracts/Payroll.sol

```diff
@@ -56,27 +58,33 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
 
     struct Employee {
         address accountAddress; // unique, but can be changed over time
-        mapping(address => uint8) allocation;
+        mapping(address => uint256) allocation;
         uint256 denominationTokenSalary; // per second in denomination Token
         uint256 accruedValue;
         uint64 lastPayroll;
         uint64 endDate;
     }
 
+    IFeed public feed;
     Finance public finance;
     address public denominationToken;
-    IFeed public feed;
     uint64 public rateExpiryTime;
 
-    mapping(uint256 => Employee) private employees;     // employee ID -> employee
-    // Employees start at index 1, to allow us to use employees[0] to check for non-existent address
```

Comment:
This is still true, right? I would keep the comment then.

Reply:
Answered [below](https://github.com/aragon/aragon-apps/pull/744#discussion_r270369503)